.github/workflows/publish-command.yml

```diff
@@ -177,6 +184,58 @@ jobs:
           SLACK_COLOR: DC143C
           SLACK_TITLE: "Failed to publish connector ${{ github.event.inputs.connector }} from branch ${{ github.ref	 }}"
           SLACK_FOOTER: ""
+      - name: Check if connector in definitions yaml
+        run: |
+          connector="airbyte/${{ env.IMAGE_NAME }}"
```

Comment:
Where's `env.IMAGE_NAME` defined?

Reply:
I found it's defined in `Prepare Sentry Release` step, but it'd be cool get it in a separate step as it's not only sentry related.

Reply:
yeah agreed, will address this